build.gradle

```diff
@@ -14,6 +14,16 @@ repositories {
     jcenter()
 }
 
+jar {
+    manifest {
+        attributes 'Main-Class': 'fr.sncf.osrd.App'
+    }
+
+    from {
+        configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
+    }
+}
+
```

Comment:
we're using https://imperceptiblethoughts.com/shadow/ to build shadow jars properly, no need for that I believe (the build target for getting a fat jar is called `shadowJar`)

Reply:
fixed by #7836f4ef7b7a68d0e324e07fba64be172937df7f